charts/artifact-registry/chart/templates/cm-nginx.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-conf
+data:
+  nginx.conf: |
+    events {
+        worker_connections  1024;
+    }
+    http {
+      server {
+          listen       80;
+          location / {
+              root   /charts;
+              index  index.yaml;
+          }
+      }
```

The indentation is off:
```suggestion
    http {
        server {
            listen       80;
            location / {
                root   /charts;
                index  index.yaml;
            }
        }
```